nodes/core/parsers/70-HTML.js

```diff
@@ -41,6 +46,11 @@ module.exports = function(RED) {
                             /* istanbul ignore else */
                             if (pay2) {
                                 msg.payload = pay2;
+				msg.parts = {
+				    id: msg._msgid,
+				    index: index,
```

Comment:
should we also add the msg.parts.type ? in case someone wants to rebuild an array automatically ?

Reply:
Thank you for reviewing.
It seems to be useful.
And msg.parts.ch is also needed for automatic join.